.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: php
 
 php:
-  - 5.3
```

Comment:
Omnipay 2 needs to retain support for 5.3 - this will be dropped/changed if upgraded to Omnipay 3

Reply:
Ok, I'll re-introduce

Reply:
If you're going to add 5.3 back into Travis, you'll need to use the method in the `.travis.yml` on master - it's only supported on a particular distribution.

Reply:
Got it